source/dhtslib/sam/md.d

```diff
@@ -152,7 +152,7 @@ debug (dhtslib_unittest) unittest
     import std.array : array;
     import std.path : buildPath, dirName;
 
-    auto bam = SAMFile(buildPath(dirName(dirName(dirName(__FILE__))), "htslib",
+    auto bam = SAMFile(buildPath(dirName(dirName(dirName(dirName(__FILE__)))), "htslib",
```

Comment:
You don't need to chagne this at this time (I am planning to extract and separate "integration tests" from "unit tests" at some point in the future), but I would instead of parent, parent, parent of __FILE__ (which could as it does in this commit, change location), just use cwd of the test binary which has `htslib` subdir as its immediate child